src/Main.java

```diff
@@ -1,9 +1,37 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
 
+        Translator translator = new Translator(alphabetic, numeric);
+        System.out.println(translator.translate(6));
         // Hier mag je je code schrijven voor de hoofd-opdracht
+        Boolean play = true;
```

Comment:
Mag primitief datatype 'boolean' zijn ipv wrapper type. Neemt minder geheugen in beslag. 'Boolean' gebruik je bijvoorbeeld als je ook 'null' waarde zou willen geven of als je de variable zou willen toevoegen aan een List of een Map